app/helpers/ems_middleware_helper/textual_summary.rb

```diff
@@ -0,0 +1,70 @@
+module EmsMiddlewareHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    %i(name type hostname port)
+  end
+
+  def textual_group_relationships
+    # Order of items should be from parent to child
+    items = []
+    items.concat(%i(middleware_servers middleware_deployments))
+    items
```

Comment:
I just realized this temporary items variable is not needed and this method can just return the array directly, much like `textual_group_properties`